tools/lister.py

```diff
@@ -37,7 +37,7 @@ def get_ftype(fpath, use_shebang):
         return ''
 
 def list_files(targets=[], ftypes=[], use_shebang=True, modified_only=False,
-               exclude=[], group_by_ftype=False):
+               exclude=[], group_by_ftype=False, exclude_flag=False):
```

Comment:
I think this is probably not the right API.  Better would be to support passing additionally a list of regular expressions on the filenames to check in addition to the simpler existing excluded files/directories.  Would call it e.g. `exclude_patterns=[]`.